### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by Keavn-CSU**
Initial pull to main

### CODE REVIEW

**Comment by scritty1249** [comment]
Looks good but you need to add a matching README.md before the merge or we're not gonna have a repository description.

File: README.md
Comment (scritty1249) [inline README.md:1]: we need to keep this

**Comment by scritty1249** [comment]
LGTM.

**Review by scritty1249 — APPROVED** [review]

**Review by W-Sammy — APPROVED** [review]
No conflicts therefore Let's Gamble, Try Merging

**Pull request merged**